lib/ansible/modules/cloud/ovirt/ovirt_template.py

```diff
@@ -157,6 +157,10 @@
             - Upper bound of template memory up to which memory hot-plug can be performed.
               Prefix uses IEC 60027-2 standard (for example 1GiB, 1024MiB).
         version_added: "2.6"
+    clone_name:
+        description:
+            - Name for importing Template from storage domain.
```

Comment:
Please write that if not specified it will use name specified by `name` parameter.